content/doc/developer/plugin-tutorial/basic-setup.adoc

```diff
@@ -0,0 +1,20 @@
+---
+title: Plugin Tutorial
```

Comment:
Is there a reason this copies & pastes `content/doc/developer/plugin-tutorial/index.adoc`?